package/ebs/rancher-ebs

```diff
@@ -305,10 +323,17 @@ detach() {
 
 mountdest() {
     local error
-
-    error=`mount $DEVICE $MNT_DEST 2>&1`
-    if [ $? -ne 0 ]; then
-        print_error $error
+    cd "${MNT_DEST}-staging"
+    if [ ! -z "$(ls -a | grep '.rancher-v2')" ]; then
```

Comment:
-e '.rancher-v2' would test if the file exists.
Would that be a cleaner solutoin here?